src/styles/blocks/page.scss

```diff
@@ -0,0 +1,16 @@
+* {
+  display: flex;
+  box-sizing: border-box;
+}
+
+head {
```

Comment:
Applying styles to the 'head' element is not recommended, as it should not be visible in the rendered page. Remove the 'head' selector and its associated styles.